src/test/cypress/support/pageobjects/AssessmentDashboard.ts

```diff
@@ -0,0 +1,31 @@
+export class AssessmentDashboard {
+    openExerciseDashboard() {
+        cy.get('[jhitranslate="entity.action.exerciseDashboard"]').should('be.visible').click();
+    }
+
+    confirmInstruction() {
+        cy.contains(/I have read and understood the instructions|Start participating in the exercise/).click();
+    }
+
+    startNewAssessment() {
+        cy.contains('Start new assessment').should('be.visible').click();
```

Comment:
Same as above. I used [this selector](https://github.com/ls1intum/Artemis/blob/778675b0f629bfe67100229484116e4a36ab38bb/src/test/cypress/support/pageobjects/assessment/ExerciseAssessmentDashboardPage.ts#L10)
Also the visibility assertion is redundant here

Reply:
done